front/tests/pages/stdcm-page-model.ts

```diff
@@ -307,68 +320,134 @@ class STDCMPage {
-    await expect(this.destinationArrival).toHaveValue('asSoonAsPossible');
+    expect(destinationCiValue).toContain(DESTINATION_DETAILS.suggestionResult);
+    // Verify Default Values
+    await expect(this.dynamicDestinationCh).toHaveValue(DESTINATION_DETAILS.defaultChValue);
+    await expect(this.destinationArrival).toHaveValue(DESTINATION_DETAILS.defaultArrivalType);
     await expect(this.warningBox).toContainText(translations.stdcmErrors.noScheduledPoint);
     await expect(this.dateDestinationArrival).not.toBeVisible();
     await expect(this.timeDestinationArrival).not.toBeVisible();
     await expect(this.toleranceDestinationArrival).not.toBeVisible();
-    await this.destinationArrival.selectOption('preciseTime');
-    await expect(this.dateDestinationArrival).toHaveValue('17/10/24');
-    await expect(this.timeDestinationArrival).toHaveValue('00:00');
-    await expect(this.toleranceDestinationArrival).toHaveValue('-30/+30');
-    await this.dateDestinationArrival.fill('18/10/24');
+    // Select 'preciseTime' and Verify Values
+    await this.destinationArrival.selectOption(DESTINATION_DETAILS.updatedArrivalType);
+    await expect(this.destinationArrival).toHaveValue(DESTINATION_DETAILS.updatedArrivalType);
+    await expect(this.dateDestinationArrival).toHaveValue(DESTINATION_DETAILS.defaultArrivalDate);
+    await expect(this.timeDestinationArrival).toHaveValue(DESTINATION_DETAILS.defaultArrivalTime);
+    await expect(this.toleranceDestinationArrival).toHaveValue(
+      DESTINATION_DETAILS.defaultTolerance
+    );
+    // Update Date and Time Values
+    await this.dateDestinationArrival.fill(DESTINATION_DETAILS.updatedDate);
+    await expect(this.dateDestinationArrival).toHaveValue(DESTINATION_DETAILS.updatedDate);
     await this.timeDestinationArrival.click();
-    await this.setHourLocator('01'); // Select hour 01
-    await this.setMinuteLocator('35'); // Select minute 35
+    await this.setHourLocator(DESTINATION_DETAILS.updatedHour);
+    await this.setMinuteLocator(DESTINATION_DETAILS.updatedMinute);
     await this.incrementButton.dblclick(); // Double-click the +1 minute button to reach 37
     await this.timeDestinationArrival.click();
-    await this.fillToleranceField('-75', '+45', false);
+    await expect(this.timeDestinationArrival).toHaveValue(DESTINATION_DETAILS.updatedTimeValue);
+
+    // Update Tolerance and Verify Warning Box
+    await this.fillToleranceField(
+      DESTINATION_DETAILS.toleranceNegative,
+      DESTINATION_DETAILS.tolerancePositive,
+      false
+    );
     await expect(this.warningBox).not.toBeVisible();
   }
 
   // Fill origin section
   async fillOriginDetailsLight() {
-    await this.dynamicOriginCi.fill('North');
+    const ORIGIN_DETAILS = {
+      originInput: 'North',
+      expectedChValue: 'BV',
+      arrivalType: 'preciseTime',
+      arrivalDate: '17/10/24',
+      arrivalTime: '20:21',
+      toleranceNegative: '-60',
+      tolerancePositive: '+15',
+    };
```

Comment:
addition to previous comments:

if `ORIGIN_DETAILS` and `DESTINATION_DETAILS` have to change, they should be a functions, so you could just return the values to use instead of declaring static and bulky consts

Reply:
I used a const.ts file to store all the values and reuse common ones